editor.planx.uk/src/@planx/components/Section/model.ts

```diff
@@ -10,3 +13,71 @@ export const parseSection = (
   title: data?.title || "",
   ...parseMoreInformation(data),
 });
+
+export function computeSectionStatuses({
```

Comment:
could you share about decision to move this function out of navigation store and into the component model instead? my first impression is that it feels out a bit out of place here!

Reply:
It's a pure function that is only being used by this one component so I thought the model makes the most sense. The store should ideally be for managing state rather than grouping/sharing functionality. That's my view anyway.